alpha/apps/kaltura/lib/myXmlUtils.php

```diff
@@ -2,41 +2,48 @@
 
 class myXmlUtils
 {
-	public static function validateXmlFileContent($filePath, $partnerId)
+	public static function validateXmlFileContent($filePath, $purifyParams = array())
 	{
-		if (!$filePath || !$partnerId)
+		if (!$filePath)
 		{
 			return true;
 		}
-
+		
 		$fileType = kFileUtils::getMimeType($filePath);
 		if (strpos($fileType, 'html') !== false || strpos($fileType, 'xml') !== false)
 		{
-			$partner = PartnerPeer::retrieveByPK($partnerId);
 			$xmlContent = kFile::getFileContent($filePath);
 			
 			$dom = new KDOMDocument();
 			$dom->loadXML($xmlContent);
 			$element = $dom->getElementsByTagName('script')->item(0);
-			if($element)
+			if ($element)
 			{
 				return false;
 			}
 			
-			if ($partner && $partner->getPurifyImageContent())
+			if ($purifyParams)
 			{
-				$modifiedContent = self::purifyField('thumbasset', 'content' , $xmlContent);
-				
-				if ($modifiedContent != $xmlContent)
-				{
-					kFile::setFileContent($filePath, $modifiedContent);
-				}
+				self::purifyXmlContent($filePath, $xmlContent, $purifyParams);
 			}
 		}
-
+		
 		return true;
 	}
 	
+	public static function purifyXmlContent($filePath, $xmlContent, $purifyParams)
+	{
+		if (!is_null($purifyParams[0]) && !is_null($purifyParams[1]))

```

Comment:
check with isset and not is_null
change $purifyParams[0] to $purifyParams['className'] and $purifyParams[1] to $purifyParams['fieldName']